src/Site.js

```diff
@@ -385,6 +412,193 @@ Site.prototype.createPage = function (config) {
   });
 };
 
+/**
+ * Converts an existing Github wiki or docs folder to a MarkBind website.
+ */
+Site.prototype.convert = function (rootPath) {
+  return new Promise((resolve, reject) => {
+    this.addIndexPage(rootPath)
+      .then(() => this.addAboutPage(rootPath))
+      .then(() => this.addFooterToDefaultLayout(rootPath))
+      .then(() => this.addSiteNavToDefaultLayout(rootPath))
+      .then(() => this.addDefaultLayoutToSiteConfig(rootPath))
+      .then(() => this.addTopNavToAddressablePages(rootPath))
+      .then(resolve)
+      .catch(reject);
+  });
+};
+
+/**
+ * Adds an index page to site if not present.
```

Comment:
From the logic is seems like `index.md` is generated only if `README.md` or `Home.md` are present which seems to contradict the comment.

Reply:
Currently, the logic is to copy over content from `README.md` or `Home.md` if they exist, to the default `index.md` that is generated during `markbind init`.
Updated the comment to reflect the current behaviour.

Reply:
@yamgent @acjh should we have some other default content in `index.md` for converted wikis/docs folders? Or is the current logic okay? 
Also, using `README.md` / `Home.md` seems a bit hacky, but I'm not sure how else to do this as of now...

Reply:
Apologies for the late reply.

We can use `README.md` as `index.md` if there isn't already an original `index.md`. This behaviour is not dissimilar to how [GitHub Pages](https://github.blog/2016-12-09-publishing-with-github-pages-now-as-easy-as-1-2-3/) do it, so if we want to mimic this behaviour, I think it is reasonable.

If neither `README.md`/`Home.md` nor `index.md` is present, which means we cannot and won't have an `index.html`, doing a `markbind serve` will cause it to present to the user a list of files and directories on the root path. To me, I think that is OK, since that way authors can easily check the content of different .html files, rather than having to look at a default index.html which he is going to change anyway. In short: Nope, probably don't need to supply a default `index.md` for that situation.

Reply:
Thanks!